Slider/Assets/Scripts/Audio/AudioManager.cs

```diff
@@ -26,7 +27,13 @@ public class AudioManager : Singleton<AudioManager>
     private static FMOD.Studio.Bus sfxBus;
     private static FMOD.Studio.Bus musicBus;
 
-    public static AudioManager instance;
+    [SerializeField]
+    List<AudioModifier> modifierPool;
+    Dictionary<AudioModifier.ModifierType, AudioModifier> modifiers;
```

Comment:
these should prob be public `static` if they can, so that we can make other methods below `static`

nit: can u add `private` or `protected` or `public` to class vars; i think it makes the code more readable bc its easier to see what the scope is and if its a class var and not local var